orion/primitives/jsons/orion.primitives.timeseries_postprocessing.build_anomaly_intervals.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I noticed `build_anomaly_intervals` is not being used. Is there a reason for adding it to the PR? (similar comment applies to the function, if it is not part of the pipelines, we can remove it)